src/main/java/com/palantir/gradle/versions/FixLegacyJavaConfigurationsPlugin.java

```diff
@@ -54,17 +55,16 @@ public final void apply(Project project) {
     }
 
     private void fixLegacyResolvableJavaConfigurations(Project project, Configuration unifiedClasspath) {
-        Stream.of(
-                        JavaPlugin.COMPILE_CONFIGURATION_NAME,
-                        JavaPlugin.COMPILE_ONLY_CONFIGURATION_NAME,
-                        JavaPlugin.RUNTIME_CONFIGURATION_NAME)
-                .map(project.getConfigurations()::named)
-                .forEach(confProvider -> confProvider.configure(conf -> {
+        // TODO(fwindheuser): Remove compile and runtime after stating to build with Gradle 7+
```

Comment:
Perhaps we should enforce this by adding a precondition that `MINIMUM_GRADLE_VERSION` < 7?

Reply:
I think this precondition would get triggered when a consuming repository builds with Gradle 7? I think for now, we want to enable users to use either Gradle 6 or 7.